sim/paladin/hand_of_reckoning.go

```diff
@@ -0,0 +1,39 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+)
+
+func (paladin *Paladin) registerHandOfReckoningSpell() {
```

Comment:
Can we start this function with a check like so:
```
if paladin.HasMajorGlyph(proto.PaladinMajorGlyph_GlyphOfReckoning) {
  return
}
```

And then later just check if `paladin.HandOfReckoning == nil` to determine the presence of the spell. This will play more nicely with future APL changes, and allow the `ret.HandOfReckoning` boolean to be removed.